packages/desktop-client/src/components/manager/ConfigServer.tsx

```diff
@@ -121,16 +143,42 @@ export function ConfigServer() {
       </Text>
 
       {error && (
-        <Text
-          style={{
-            marginTop: 20,
-            color: theme.errorText,
-            borderRadius: 4,
-            fontSize: 15,
-          }}
-        >
-          {getErrorMessage(error)}
-        </Text>
+        <>
+          <Text
+            style={{
+              marginTop: 20,
+              color: theme.errorText,
+              borderRadius: 4,
+              fontSize: 15,
+            }}
+          >
+            {getErrorMessage(error)}
+          </Text>
+          {isElectron() && (
```

Comment:
Only shown on Electron - browsers don't have the issue. Node has to explicitly trust the cert, that only affects Electron.